Classes/Controller/ExportController.php

```diff
@@ -187,8 +187,8 @@ protected function replaceExtensionKey($extensionKey, $string)
         $lowercaseExtensionKey = strtolower($camelCasedExtensionKey);
 
         $string = preg_replace(
-            '/(\s+|\'|,|.)(tx_)?mask_/',
-            '\\1\\2' . $lowercaseExtensionKey . '_',
+            '/(\s+|\'|,|.)(tx_)?mask/',
```

Comment:
please use a proper regular expression like '/(\s+|\'|,|.)(tx_)?mask(_|\.)/' and its replacement '\\1\\2' . $lowercaseExtensionKey . '\\3'